galaxy/wrapper/plotHeatmap.xml

```diff
@@ -228,6 +238,10 @@
                     help="The default is to make one plot per bigWig file, i.e., all samples next to each other. Choosing this option will make one plot per group of regions."/>
 
                 <expand macro="kmeans_clustering" />
+                <param argument="--clusterUsingSamples" type="text" size="30"
+                    label="List of samples to be used for clustering"
+                    help="List of sample numbers (order as in matrix), which are used by --kmeans or --hclust for clustering.
+                    If no value is set, it uses all samples. Example: 1 3 (space separated!)">
```

Comment:
sanitizer should be added

Reply:
@bgruening I have already changed them. Is it good now? If so, I am gonna merge it.